test/test_core.py

```diff
@@ -5686,7 +5686,7 @@ def test_stat_chmod(self):
       self.skipTest('mode bits work differently on windows')
     if nodefs and self.get_setting('WASMFS'):
       self.skipTest('test requires symlink creation which currently missing from wasmfs+noderawfs')
-    self.do_runf('stat/test_chmod.c', 'success')
+    self.do_runf('stat/test_chmod.c', 'success', cflags=['-Werror=conversion'])
```

Comment:
Makes me wonder if we should make an effort to enable this in whole test suite.

We currently enable `-Werror` by default: https://github.com/emscripten-core/emscripten/blob/4cab0399ccb663ccacc3d5b15fa765a8ec651b8e/test/common.py#L1202

But we don't enable `-Wall` (which looks like it doesn't actually include `-Wconversion` anyway )